ceph/rados/core_workflows.py

```diff
@@ -2304,114 +2304,73 @@ def warning_check(timeout):
         log.info("Daemon crash warning found in ceph health")
         return True
 
-    def add_network_delay_on_host(
-        self, hostname, delay="20ms", packet_loss="0.1", set_delay=True
-    ) -> bool:
+    def configure_host_as_client(self, host_node):
         """
-        This method makes use of tc utility to introduce network delays and packet drops on the nodes.
-
-        Args::
-            hostname: Name of the host
-            delay: delay to be added on the host
-            packet_loss: percentage of packets to be dropped
-            set_delay: If true, sets the network delay, otherwise removes the delay added on the host
-
-        Return:
-            Pass -> True
-            Fail -> False
+        Purpose of this module is to configure the ceph keyring and conf
+        on a cluster host which is not installer to run ceph commands from
+        it
+        Args:
+            host_node: node object of the host which needs to be converted
+        Returns: None
         """
-        host_nodes = self.ceph_cluster.get_nodes()
-        host_obj = None
-        for node in host_nodes:
-            if (
-                re.search(hostname, node.hostname)
-                or re.search(hostname, node.vmname)
-                or re.search(hostname, node.shortname)
-            ):
-                host_obj = node
-        if not host_obj:
-            log.error(f"Host object for host {hostname} could not be found")
-            return False
+        # copy /etc/ceph/ files to input host
+        ceph_conf, _ = self.node.shell(["cat /etc/ceph/ceph.conf"])
+        ceph_keyring, _ = self.node.shell(["cat /etc/ceph/ceph.keyring"])
+        host_node.exec_command(sudo=True, cmd="mkdir -p /etc/ceph")
 
-        # Checking and installing iproute package on node
+        for cont in [ceph_conf, ceph_keyring]:
+            file_name = (
+                "/etc/ceph/ceph.conf" if cont == ceph_conf else "/etc/ceph/ceph.keyring"
+            )
+            file_ = host_node.remote_file(sudo=True, file_name=file_name, file_mode="w")
+            file_.write(cont)
+            file_.flush()
+            file_.close()
+
+        # Checking and installing ceph-common package on host
         try:
-            host_obj.exec_command(sudo=True, cmd="rpm -qa | grep iproute")
+            out, rc = host_node.exec_command(
+                sudo=True, cmd="rpm -qa | grep ceph-common"
+            )
         except Exception:
-            host_obj.exec_command(sudo=True, cmd="yum install iproute -y")
-        log.debug("IProute package is present on the host")
+            host_node.exec_command(sudo=True, cmd="yum install -y ceph-common")
 
-        log.info(
-            "Getting the network interface on the host to add network delay & packet loss"
-        )
-        interface = host_obj.search_ethernet_interface(host_nodes)
-        log.debug(f"Fetched interface for host is : {interface}")
+    def get_osd_fsid(self, osd_id):
```

Comment:
Lets update the name to `get_osd_uuid`. Fsid is unique and general for entire cluster. might create confusion in future.

Reply:
The commands in the administration guide mentioned as-
1. ceph-volume lvm new-db --osd-id OSD_ID --osd-fsid OSD_FSID --target VOLUME_GROUP_NAME/LOGICAL_VOLUME_NAME
2. ceph-volume lvm migrate --osd-id OSD_ID --osd-fsid OSD_UUID --from data --target VOLUME_GROUP_NAME/LOGICAL_VOLUME_NAME

Anyway, I'll change the name.